HW_5/task_28.py

```diff
@@ -0,0 +1,18 @@
+"""""""""""
+Задача 28: Напишите рекурсивную функцию sum(a, b), возвращающую сумму двух целых неотрицательных чисел. Из всех арифметических операций допускаются только +1 и -1. 
+Также нельзя использовать циклы.
+*Пример:*
+2 2
+    4
+"""""
+
+def sum(a,b):
+    result = 0
+    if b == 0:
+        return a
+    elif b > 0:
+        result = a + sum(1, b - 1)
+        return result
```

Comment:
В это условии нет смысла
```suggestion
    return a + sum(1, b - 1)
```